test/explorer.html

```diff
@@ -74,6 +74,13 @@ <h1>Content Explorer (no folder upload)</h1>
                 const { ContentExplorer } = Box;
                 const token = document.querySelector('.token').value || localStorage.getItem('token');
                 const folderId = document.querySelector('.folder').value || localStorage.getItem('folder');
+                const features = {
```

Comment:
Is the goal to keep the grid view hidden? If that's the case, then I think leaving the features object out entirely should keep it hidden. Though maybe you could add a separate explorer example with the feature turned on?